src/server/scripts/Pandaria/CMakeLists.txt

```diff
@@ -11,20 +11,62 @@
+  Pandaria/WorldBosses/boss_galion.cpp
+  Pandaria/WorldBosses/boss_nalak.cpp
+  Pandaria/WorldBosses/boss_oondasta.cpp
+  Pandaria/WorldBosses/boss_sha_of_anger.cpp
+  #Zone Scripts
+  Pandaria/Zones/zone_dread_wastes.cpp
+  Pandaria/Zones/zone_isle_of_giants.cpp
+  Pandaria/Zones/zone_isle_of_thunder.cpp
+  Pandaria/Zones/zone_krasarang_wilds.cpp
+  Pandaria/Zones/zone_kun_lai_summit.cpp
+  Pandaria/Zones/zone_the_jade_forest.cpp
+  Pandaria/Zones/zone_the_veiled_stair.cpp
+  Pandaria/Zones/zone_townlong_steppes.cpp
+  Pandaria/Zones/zone_vale_of_eternal_blossoms.cpp
+  Pandaria/Zones/zone_valley_of_the_four_winds.cpp
+  #Sub-Zone TimelessIsle
+  Pandaria/Zones/TimelessIsle/boss_chi_ji.cpp
+  Pandaria/Zones/TimelessIsle/boss_niuzao.cpp
+  Pandaria/Zones/TimelessIsle/boss_ordos.cpp
+  Pandaria/Zones/TimelessIsle/boss_xuen.cpp
+  Pandaria/Zones/TimelessIsle/boss_yu_lon.cpp
+  Pandaria/Zones/TimelessIsle/timeless_isle.h
+  Pandaria/Zones/TimelessIsle/timeless_isle_global.cpp
+  #Rares Sub-Zone TimelessIsle
+  Pandaria/Zones/TimelessIsle/Rares/rare_mob_archereus_of_flame.cpp
+  Pandaria/Zones/TimelessIsle/Rares/rare_mob_bufo.cpp
+  Pandaria/Zones/TimelessIsle/Rares/rare_mob_champion_of_the_black_flame.cpp
+  Pandaria/Zones/TimelessIsle/Rares/rare_mob_chelon.cpp
+  Pandaria/Zones/TimelessIsle/Rares/rare_mob_cinderfall.cpp
+  Pandaria/Zones/TimelessIsle/Rares/rare_mob_cranegnasher.cpp
+  Pandaria/Zones/TimelessIsle/Rares/rare_mob_dread_ship_vazuvius.cpp
+  Pandaria/Zones/TimelessIsle/Rares/rare_mob_emerald_gander.cpp
+  Pandaria/Zones/TimelessIsle/Rares/rare_mob_evermaw.cpp
+  Pandaria/Zones/TimelessIsle/Rares/rare_mob_flintlord_gairan.cpp
+  Pandaria/Zones/TimelessIsle/Rares/rare_mob_garnia.cpp
+  Pandaria/Zones/TimelessIsle/Rares/rare_mob_golganarr.cpp
+  Pandaria/Zones/TimelessIsle/Rares/rare_mob_great_turtle_furyshell.cpp
+  Pandaria/Zones/TimelessIsle/Rares/rare_mob_gu_chi_the_swarmbringer.cpp
+  Pandaria/Zones/TimelessIsle/Rares/rare_mob_huolon.cpp
+  Pandaria/Zones/TimelessIsle/Rares/rare_mob_imperial_python.cpp
+  Pandaria/Zones/TimelessIsle/Rares/rare_mob_ironfur_steelhorn.cpp
+  Pandaria/Zones/TimelessIsle/Rares/rare_mob_jakur_of_ordon.cpp
+  Pandaria/Zones/TimelessIsle/Rares/rare_mob_karkanos.cpp
+  Pandaria/Zones/TimelessIsle/Rares/rare_mob_leafmender.cpp
+  Pandaria/Zones/TimelessIsle/Rares/rare_mob_monstrous_spineclaw.cpp
+  Pandaria/Zones/TimelessIsle/Rares/rare_mob_rattleskew.cpp
+  Pandaria/Zones/TimelessIsle/Rares/rare_mob_rock_moss.cpp
+  Pandaria/Zones/TimelessIsle/Rares/rare_mob_spelurk.cpp
+  Pandaria/Zones/TimelessIsle/Rares/rare_mob_spirit_of_jadefire.cpp
+  Pandaria/Zones/TimelessIsle/Rares/rare_mob_stinkbraid.cpp
+  Pandaria/Zones/TimelessIsle/Rares/rare_mob_tsavo_ka.cpp
+  Pandaria/Zones/TimelessIsle/Rares/rare_mob_urdur_the_cauterizer.cpp
+  Pandaria/Zones/TimelessIsle/Rares/rare_mob_watcher_osu.cpp
+  Pandaria/Zones/TimelessIsle/Rares/rare_mob_zesqua.cpp
+  Pandaria/Zones/TimelessIsle/Rares/rare_mob_zhu_gon_the_sour.cpp
```

Comment:
rares could be scripted in SAI, but if you want to script them in c++ its fine, but adding unfinished placeholder scripts is definitely the wrong idea script them first

Reply:
The Tempholders should exist for references. i am not the only one here who spend time into skyfire as thanks i get asskicks and admin remove. so let other peoples do some stuff...

And SAI Yes ... 
![screenshot_1](https://user-images.githubusercontent.com/10685816/37236878-c77ca8bc-240c-11e8-8e35-eef2f21b2159.png)